ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/133__embedded_operator.robot

```diff
@@ -0,0 +1,153 @@
+*** Settings ***
+Library    String
+Library    OpenShiftLibrary
+Library    OperatingSystem
+Library    ../../../../../libs/Helpers.py
+Resource   ../../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource   ../../../../../tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot
+Resource   ../../../../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+Suite Teardown    Uninstall RHODS V2 Embedded
+
+
+***Variables***
+${cluster_type}              selfmanaged
+${image_url}                 ${EMPTY}
+${dsci_name}                 default
+${dsc_name}                  default
+
+
+*** Test Cases ***
+Install "Embedded Operator" RHODS
+    [Documentation]
+    [Tags]
+    Set Global Variable    ${UPDATE_CHANNEL}    embedded
+    Install Embedded RHODS    ${cluster_type}     ${image_url}
+
+Verify Embedded RHODS Installation
+    [Documentation]
+    [Tags]
+    RHODS Embedded Verification
+
+
+*** Keywords ***
+Install Embedded RHODS
```

Comment:
why not implementing in tasks like done for uninstall keyword?

Reply:
Ah I see now there are 2 keywords to delete RHODS v2, one for embedded channel explicitly. Could we merge the 2 kw?